packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/form.tsx

```diff
@@ -0,0 +1,133 @@
+    onChangeInputValue,
+    onHideDropdownList,
+    onScrollStopDropdownList,
+    symbol_dropdown,
+    trade_type_dropdown,
+    is_mobile,
+    selected_symbol,
+    selected_trade_type,
+    selected_duration_unit,
+    setCurrentFocus,
+    selected_type_strategy,
+    description,
+    setActiveTab,
+}: TQuickStrategyForm) => {
+    const { min, max } = selected_duration_unit;
+
+    const setDefaultValidationNumber = () =>
+        Yup.number()
+            .typeError(localize('Should be a number'))
+            .round('ceil')
+            .min(1, localize('Must be a number higher than 0'));
+
+    const SchemaFields = Yup.object().shape({
+        'quick-strategy__duration-value': Yup.number()
+            .typeError(localize('Should be a number'))
+            .required(localize('Field cannot be empty'))
+            .min(min, localize('Minimum duration: {{ min }}', { min }))
+            .max(max, localize('Maximum duration: {{ max }}', { max })),
+
+        'quick-strategy__stake': setDefaultValidationNumber(),
+        'quick-strategy__loss': setDefaultValidationNumber(),
+        'martingale-size': setDefaultValidationNumber().min(3, localize('Must be a number higher than 2')),
+        'alembert-unit': setDefaultValidationNumber(),
+        'oscar-unit': setDefaultValidationNumber(),
+        'quick-strategy__profit': setDefaultValidationNumber(),
+    });
+    return (
+        <Formik
+            initialValues={initial_values}
+            validationSchema={SchemaFields}
+            onSubmit={createStrategy}
+            enableReinitialize={true}
+            validateOnMount={true}
+        >
+            {({
+                errors,
+                handleChange,
+                values,
+                isSubmitting,
+                setFieldValue,
+                submitForm,
+            }: FormikProps<TQuickStrategyFormValues>) => {
+                const is_valid =
+                    Object.keys(errors).length === 0 && !Object.values(values).some(elem => (elem as string) === '');
+                const is_submit_enabled = !isSubmitting && is_valid;
+
+                return (
+                    <Form
+                        className={classNames('quick-strategy__form', {
+                            'quick-strategy__form--active-keyboard': is_onscreen_keyboard_active,
+                        })}
+                    >
+                        <ThemedScrollbars height='535px' autohide is_bypassed={is_mobile}>
+                            <div
+                                className={classNames('quick-strategy__form-content', {
+                                    'quick-strategy__form-content--active-keyboard': is_onscreen_keyboard_active,
+                                    'quick-strategy__form-content--safari-fix': isSafari(),
+                                })}
+                            >
+                                <div className='quick-strategy__title'>{localize('Quick strategy')}</div>
```

Comment:
isn't it better to use <Text> component as a wrapper for localize so we can pass props to it for `font_weight`, `line_height`, ... instead of defining them in css?
```suggestion
                                <div className='quick-strategy__title'><Text ...props>{localize('Quick strategy')}</Text></div>
```

Reply:
done ✅

Reply:
@mahdiyeh-fs thank you, I've started to use it👌
done✅